### PR TITLE
lint: no-member is usually an error

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: raiden/raiden_service.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def __init__(
 
         self.tokennetworkids_to_connectionmanagers = dict()
         self.identifier_to_results: typing.Dict[
-            typing.PaymentIdentifier,
+            typing.PaymentID,
```
Comment (hackaugusto) [inline raiden/raiden_service.py]: the linter caught this

File: raiden/utils/solc.py [hunk 2]
```diff
@@ -100,47 +95,3 @@ def compile_files_cwd(*args, **kwargs):
...
-def validate_solc():
-    if get_solc_version() is None:
-        raise RuntimeError(
-            "Couldn't find the solc in the current $PATH.\n"
-            "Make sure the solidity compiler is installed and available on your $PATH.",
-        )
-
-    try:
-        compile_files(
-            [CONTRACT_MANAGER.get_contract_path(CONTRACT_HUMAN_STANDARD_TOKEN)],
```
Comment (hackaugusto) [inline raiden/utils/solc.py]: and this

**Comment by hackaugusto** [comment]
> I wish we could work on enabling all pylint at some point in the future, enabled it once by mistake in vim, couldn't work from the amount of warnings, we could improve on this

I have it enable on vim, that is how I got this mistake. But ATM we have too many false-positives to use it in the build.

**Comment by LefterisJP** [comment]
Enabling all of them would be too much work due to the false positives, but baby steps forward like this are great! Thank you.

**Pull request merged**